js/main.js

```diff
@@ -1,28 +1,57 @@
 var entries = $("#entries");
-
 var form = $("form");
 var inputBox = $("#newEntry");
-
 var totalEl = $("#total");
 var total = 0.00;
 
 form.on("submit", updateRegister);
 
-function updateRegister() {
-  event.preventDefault();
-
-  // get the amount and clear the box
-  var amount = parseFloat(inputBox.val());
-  inputBox.val("");
-
-  // make the new element and add it to the list of entries
-  var newEl = $("<tr>");
-  newEl.append("<td></td>");
-  newEl.append("<td>" + "$" + amount.toFixed(2) + "</td>");
-
-  entries.append(newEl);
-
-  // update the total
-  total = total + amount;
-  totalEl.text("$" + total.toFixed(2));
+function Register() {
+  this.domElement = null;
+  this.total = 0.00;
+  this.initialize = function(container){
+    this.domElement = $('<div class="register"></div>');
+    container.append(this.domElement);
+  };
+  this.render = function(){
```

Comment:
Right now every time you create a new Register it will create a new copy of all of these functions as well, this could be avoided by converting these into prototype functions
```js
Register.prototype.render = function() {
```